internal/cmd/vet.go

```diff
@@ -448,14 +448,7 @@ func (c *checker) fetchDatabaseUri(ctx context.Context, s config.SQL) (string, f
 }
 
 func (c *checker) DSN(dsn string) (string, error) {
```

Comment:
Don't really need this func anymore

Reply:
Agreed, but it was easier to keep it around